README.md

```diff
@@ -137,6 +136,28 @@ sink example for demonstrating the vast amount of APIs and features in Java EE,
 we do use a very representative set. You'll find that you'll learn a fair amount
 by simply digging into the code to see how things are implemented.
 
+## Exploring the tests
+
+Cargo Tracker's testing is done using Junit 4 and Arquillian. The Arquillian configuration
+uses a [remote container](http://arquillian.org/arquillian-core/#_containers)
+such as Payara 4.1 . Therefore, to perform a test you will need to make sure
+to have a container running. 
+
+### Testing locally with Payara
+For testing locally you will first need to run a Payara 4+ server.
+
+You can do that with the following script:
+```shell script
+wget https://repo1.maven.org/maven2/fish/payara/distributions/payara/4.1.2.181/payara-4.1.2.181.zip
+unzip payara-4.1.2.181.zip && cd payara41/bin
+./asadmin start-domain
+```
+
+Now for running the tests: 
+```shell script
+mvn -Ppayara -DskipTests=false test
+```
+
 Known Issues

```

Comment:
I think these are not issues with Payara. We can just remove this section.

Reply:
I'd prefer to leave it. There are no references about how to run the tests and by default Surefire is skipping them. This might be not intuitive for a new developer.

Reply:
OK.